test/Mono.Linker.Tests.Cases/Substitutions/RemoveBody.cs

```diff
@@ -0,0 +1,65 @@
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.Substitutions
+{
+	[SetupLinkerSubstitutionFile ("RemoveBody.xml")]
+	public class RemoveBody
+	{
+		public static void Main()
+		{
+			new RemoveBody ();
+            new NestedType (5);
```

Comment:
Indent looks messed up in a couple places in this file.

Reply:
Fixed